app/ydoc-shared/src/ast/__tests__/ensoMarkdown.test.ts

```diff
@@ -91,6 +104,19 @@ test.each([
     ],
     not: ['Document', ['OrderedList', ['ListItem', ['ListMark', '1.'], ['Paragraph', 'Numbered']]]],
   },
+  /*
+  { // FIXME
+    source: '# *Formatted header*',
+    expected: [
+      'Document',
+      [
+        'ATXHeading1',
+        ['HeaderMark', '# '],
+        ['Emphasis', ['EmphasisMark', '*'], ['EmphasisMark', '*']],
+      ],
+    ],
+  },
+   */
```

Comment:
What is to be fixed here? Do we have a task for it?

Reply:
Fixed it.